warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -534,6 +542,18 @@ public void run() {
         log.info("Bulk map file loader shutting down.");
     }
     
+    protected void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
```

Comment:
I don't think this will work if the srcHdfs is different than the destHdfs, in which case the directory will have been discp'ed to the destHdfs prior to having been processed.  See the routine below that marks the directory for cleanup which works with the destHdfs.

Reply:
true!